File: web/src/models/DTOs/Contabil/DadoContabilDTO.ts
export interface DadoContabilDTO {
  idDadoContabil: number;
  contaContabil: string;
  contaContabilDescricao: string;
  itemContabil: string;
  itemContabilDescricao: string;
  natureza: string;
  naturezaDescricao: string;
  idEntidade: number;
  idOperadorInclusao: number;
  dataInativacao: string | null;
}

File: webservices/src/main/java/fieg/modulos/cadastro/dadocontabil/dto/DadoContabilDTO.java
package fieg.modulos.cadastro.dadocontabil.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class DadoContabilDTO {

    private Integer idDadoContabil;
    private String contaContabil;
    private String contaContabilDescricao;
    private String itemContabil;
    private String itemContabilDescricao;
    private String natureza;
    private String naturezaDescricao;
    private Integer idEntidade;
    private Integer idOperadorInclusao;
    private Integer idOperadorAlteracao;
    private LocalDateTime dataInclusao;
    private LocalDateTime dataAlteracao;
    private LocalDateTime dataInativacao;



}
